backends/xnnpack/quantizer/xnnpack_quantizer.py

```diff
@@ -16,25 +16,25 @@
     propagate_annotation,
     QuantizationConfig,
 )
-from torch.ao.quantization.fake_quantize import (
+from torchao.quantization.pt2e.fake_quantize import (
```

Comment:
nit: there are some affine quantization stuff which assumes torchao isn't always available, can we fix that with this? [here](https://github.com/pytorch/executorch/blob/96c10bb34f349748201905ef44d62da46fee2612/backends/xnnpack/partition/config/quant_affine_configs.py#L36-L41) is one example.

cc @mcr229